build.gradle

```diff
@@ -0,0 +1,34 @@
+buildscript {
+	ext {
+		springBootVersion = '1.5.10.RELEASE'
+	}
+	repositories {
+		mavenCentral()
+	}
+	dependencies {
+		classpath("org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}")
+	}
+}
+
+apply plugin: 'java'
+apply plugin: 'eclipse'
```

Comment:
Do we need eclipse plugin? maybe we should change it to idea plugin

Reply:
ok, I'll check if it is needed.